src/OrchardCore.Modules/OrchardCore.Liquid/OrchardCore.Liquid.csproj

```diff
@@ -10,10 +10,28 @@
     <PackageTags>$(PackageTags) OrchardCoreCMS</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Remove="wwwroot\monaco\liquid-intellisense.ts" />
+    <EmbeddedResource Remove="wwwroot\monaco\monaco.d.ts" />
+    <EmbeddedResource Remove="wwwroot\monaco\tsconfig.json" />
+  </ItemGroup>
```

Comment:
Maybe better to use an `Assets` folder that is automatically excluded from embedded resources

And an `Assets.json` to be part of the gulp pipeline that will output the right files under wwwroot

As done in other modules, and then remove these extra ItemGroup

Reply:
i left it this way because it was easier to just tsc --watch and debug :)